src/app/nft/nft-create/nft-create.component.html

```diff
@@ -0,0 +1,73 @@
+<form id="bid-form" (ngSubmit)="onSubmit()" #form="ngForm">
+  <fieldset>
+
+
+    <!--  title-->
+    <div class="form-group" >
+      <label class="control-label" for="title">Title*</label>
+      <input id="title" name="title" type="text" class="form-control" required
+             [(ngModel)]="nft.title" #title="ngModel">
+      <div class="invalid-feedback">A title is required</div>
+    </div>
+
+    <!-- description -->
+
+    <div class="form-group" >
+      <label class="control-label" for="description">Description</label>
+      <input id="description" name="description" type="text" class="form-control"
+             [(ngModel)]="nft.description" #description="ngModel">
+      <div class="invalid-feedback">A description is required</div>
+    </div>
+
+    <!-- keywords -->
+
+    <div class="form-group" >
+      <label class="control-label" for="keywords">Keywords</label>
+      <input id="keywords" name="keywords" type="text" class="form-control"
+             [(ngModel)]="nft.keywords" #keywords="ngModel">
```

Comment:
`nft.keywords` is a list, here it will be treated as a string and the API will not accept a string in this field. You could disable this field and leave it for a future sprint.